app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt

```diff
@@ -153,6 +153,7 @@ class FavoriteFragment : Fragment() {
     private fun openSearchResult(time: String) {
         SearchResultsFragmentArgs.Builder()
             .setQuery("")
+            .setType("Anything")
```

Comment:
Why hardcoding the type there's no point of chips if you're doing this you should make them as they were supposed to work instead of making a dirty fix

Reply:
@aggarwalpulkit596  I don't really get your point... So the problem is simply because the type in SearchResultFragment is set to null when being navigated from this, so results are not fetched. If I don't put the default type ("Anything") here, then sure I can put it in the SearchResultFragment, and that's pretty much the same